contrib/senpai/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "senpai"
+pkgver = "0.3.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["./cmd/senpai"]
+hostmakedepends = ["gmake", "go", "scdoc"]
+pkgdesc = "IRC client that works best with bouncers"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "ISC"
+url = "https://git.sr.ht/~taiite/senpai"
+source = f"https://git.sr.ht/~taiite/senpai/archive/v{pkgver}.tar.gz"
+sha256 = "c02f63a7d76ae13ed888fc0de17fa9fd5117dcb3c9edc5670341bf2bf3b88718"
+options = ["!debug"]
+
+def post_build(self):
+    self.do("gmake", "doc")
```

Comment:
Or I could just run `scdoc` directly to remove the `gmake` dependency if you want (the command fails with `bmake`).

Reply:
this is fine as is